code/style.css

```diff
@@ -1,7 +1,322 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body { 
+  margin: 0.1px;
+  }
+
+header{
+    display: flex;
+    flex-direction: column;
+    justify-content:start;
+    padding-bottom: 20em;
+    text-align: center;
+  }
+
+.hero-title{
+    font-family: 'Forum', cursive;
+    letter-spacing: 10px;
+    text-align: center;
+    margin-bottom: 10em;
+    color: white;
+
+  }
+  /* This is the text box ontop of the header video */
+
+.hero-text-box {
+  width: 90%;
+  margin-left: 4em;
+  max-width: 800px;
+}
+.h2 {
+  color: whitesmoke;
+  font-family: 'Work Sans', sans-serif;
+  text-align: left;
+  font-size: 40px;
+  width: 60%;
+  padding-left: 1.75em;
+  letter-spacing: 0.10em;
+}
+
+.header-text{
+  color: whitesmoke;
+  font-family: 'Work Sans', sans-serif;
+  text-align: left;
+  width: 60%;
+  line-height: 1.5em;
+  padding-left: 3em;
+  font-size: 20px;
+}
+
+video#bgvideo {
+  position:fixed;
+  top: 50%;
+  left: 50%;
+  min-width: 100%;
+  min-height: 100%;
+  width: auto;
+  height: auto;
+  z-index: -100;
+  -ms-transform: translateX(-50%) translateY(-50%);
+  -moz-transform: translateX(-50%) translateY(-50%);
+  -webkit-transform: translateX(-50%) translateY(-50%);
+  transform: translateX(-50%) translateY(-50%);
+  /* background: url(placeholder.jpg) no-repeat; */
+  background-size: cover;
+}
+
+/* This is to create space before the forms and help with the free-flow feeling */
+.none{
+  padding: 2em;
+  background-color: transparent;
+  }
+
+/* Section underneath the form and the accordion   */
+.two{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: nowrap; 
+  gap: 3em;
+  padding: 1em 7em 10em 7em;
+  background-color: transparent;
+}
+
+.form{
+  border-style: none;
+  padding: 2em;
+  text-align: left;
+  background-color: #B8CABF;
+  border-radius: 0.5em;
+  font-family: 'Work Sans', sans-serif;
+  opacity: 80%;
+}
+.form:hover{
+  opacity: 100%;
+}
+
+input{
+  border: solid;
+  border-width: 1px;
+  font-size: 1em;
+  padding: 1em;
+  margin-bottom: 1em;   
+}
+
+.textblock{
+  width: 90%;
+}
+
+input:valid{
+  box-shadow:0.25em 0.25em 0.25em rgb(76, 205, 76);
+}
+
+input:invalid{
+  box-shadow: 0.2em 0.2em 0.2em rgb(232, 89, 89);
+}
+
+.label {
+  display: block;
+  padding: 0.5em;
+  text-align: left;
+}
+.name{
+  border-style: solid;
+  text-align: left;
+}
+
+.button{
+  margin: 1em;
+  padding: 1em;
+  border-radius: 1em;
+  background-color: #F5D081;
+}
+
+.accordion{
+  border-style: solid;
+  padding: 2em;
+  width: 70%;
+  font-family: 'Work Sans', sans-serif;
+  font-size: 25px;
+  background-color: #B8CABF;
+  border-radius: 0.5em;
+  opacity: 80%;
+}
+
+.accordion:hover{
+  opacity: 100%;
```

Comment:
I love this too! super smart! Small changes but big impact!

Reply:
Right! I like that you appreciate it.